lib/active_model/serializer.rb

```diff
@@ -476,9 +479,7 @@ def scope
 
     def _serializable_hash
       return nil if @object.nil?
-      @node = attributes
```

Comment:
@kieran I'm okay to merge then, since I'm trusting you, here :)  It looks like you have run the specs locally, but you say they fail intermittently for you?